src/node-server/openfin-config/src/config/app.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @typescript-eslint/ban-ts-ignore */
```

Comment:
Now that we have the OpenFin config here, how do we ensure compatibility between this location and the OpenFin config that exists in the client project and is used for `npm run openfin:local` etc.?

It is worth thinking about how to keep the two in sync effectively, and the developer workflow. E.g. should the base OpenFin config file come from a shared location? Maybe we can have template JSON files that we read in and then use variable substitution to put in the environment-specific bits, instead of outputting the whole object via TypeScript?

Not sure, just thinking out loud.

Reply:
I like the out loud thinking :-) Let me some not out lout thinking and I'll come back to you.